far/FarEng.hlf.m4

```diff
@@ -5371,6 +5386,15 @@ Left/Right и Num4/Num6 действуют по-разному в зависим
 
     Изменение этого параметра возможно через ~far:config~@FarConfig@
 
+@Panel.Layout.DoubleGlobalColumnSeparator
```

Comment:
Could it be an alias to @PanelSettings topic please?

Reply:
Sure. Done.